portable_mds/template/mds.py

```diff
@@ -16,9 +16,9 @@ def __init__(self, config, version=1):
         self.version = version
 
     def reset_caches(self):
-        self._RUNSTART_CAHCE.clear()
-        self._RUNSTOP_CAHCE.clear()
-        self._DESCRIPTOR_CAHCE.clear()
+        self._RUNSTART_CACHE.clear()
```

Comment:
This was also fixed upstream at https://github.com/NSLS-II/metadatastore/pull/235